src/test/java/core/basesyntax/service/parser/ParserImplTest.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax.service.parser;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
```

Comment:
Probably we can statically import methods for better readability like this

`import static org.junit.jupiter.api.Assertions.assertEquals;`

Reply:
Still exist.

Reply:
Make it everywhere.